lrpar/src/lib/ctbuilder.rs

```diff
@@ -171,6 +215,13 @@ where
         self
     }
 
+    /// Set if the generated module should be public. This is only necessary if
```

Comment:
Save comment about the second sentence as above.

Reply:
I was trying to capture your initial assertion about this not being necessary if the macro was used. Could you suggest alternative verbiage?

Reply:
I don't think I said (or, at least, I didn't mean!) that it's not necessary, but that it's not possible. You can't write `pub lrlex_mod!` for example.

I think a doc string such as "Set the visibility of the generated module to `vis` (e.g. public). Defaults to `Visibility::Private`." should be OK?